java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -12898,4 +12898,72 @@ public void wait_timeout_check(BaseClient client) {
                     () -> testClient.wait(100L, 0L).get(1000, TimeUnit.MILLISECONDS));
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xinfoStream(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String groupName = "grou1" + UUID.randomUUID();
+        String consumer = "consumer" + UUID.randomUUID();
+        String streamId0_0 = "0-0";
+        String streamId1_0 = "1-0";
+        String streamId1_1 = "1-1";
+
+        // Setup: add stream entry, create consumer group and consumer, read from stream with consumer
+        final LinkedHashMap<String, String> dataToAdd =
+                new LinkedHashMap<>() {
+                    {
+                        put("a", "b");
+                        put("c", "d");
+                    }
+                };
+        assertEquals(
+                streamId1_0,
+                client.xadd(key, dataToAdd, StreamAddOptions.builder().id(streamId1_0).build()).get());
+        assertEquals(OK, client.xgroupCreate(key, groupName, streamId0_0).get());
+
```

Comment:
you're missing a call to xreadgroup here that we have in the Python test, that will allow you to uncomment the assertions associated with the "pending" key below